.github/workflows/ci.yaml

```diff
@@ -0,0 +1,97 @@
+# yamllint disable rule:line-length
+# yamllint disable rule:braces
+
+name: Continuous Integration
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 2.15
+      - 2.16
+      - 3.0
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.operating-system }}
```

Comment:
Let's move this down in the file - it's quite surprising that we are using the value from matrix here when it's not yet defined (because people will read the file from top to bottom).

Reply:
Done.